tests/org.eclipse.passage.lic.base.tests/src/org/eclipse/passage/lic/base/tests/requirements/LicensingRequirementsTest.java

```diff
@@ -49,9 +49,13 @@ public void testExtractFromPropertiesPositive() {
 
 	@Test
 	public void testExtractFromCapabilityNegative() {
-		assertNull(LicensingRequirements.extractFromCapability(null, null, null, null, null));
-		assertNull(LicensingRequirements.extractFromCapability(null, null,
-				Collections.singletonMap(LicensingNamespaces.CAPABILITY_LICENSING_FEATURE, new Object()), null, null));
+		LicensingRequirement allNulls = LicensingRequirements.extractFromCapability(null, null, null, null, null);
+		assertEquals(LicensingProperties.LICENSING_RESTRICTION_LEVEL_ERROR, allNulls.getRestrictionLevel());
+		assertEquals(LicensingNamespaces.CAPABILITY_LICENSING_MANAGEMENT, allNulls.getFeatureIdentifier());
+		LicensingRequirement invalidAttributes = LicensingRequirements.extractFromCapability(null, null,
```

Comment:
There two separate tests here. Can we separate 'em to different test methods?

Reply:
Dobe